### DISCUSSION TITLE
HTTP Integration

### DISCUSSION THREAD

**Discussion by DaRealFrost** [cpvalente/ontime#1197] [Q&A] [answered]
So when you try bind a cycle to API, it shows {{timer.duration}} as the default, But I cannot find the docs anywhere of what the entire object looks like. I don't know if it's possible but I want to get custom fields of the current event. For context I want to integrate this into my OBS workflow.

**Comment by cpvalente** [comment, marked as answer]
Hi @DaRealFrost , thank you for taking the time to reach out.

I appreciate that the [documentation](https://docs.getontime.no/api/integrations/#using-variables-in-integrations) and [the examples of the data object](https://docs.getontime.no/api/data/runtime-data) here is maybe hard to read, I will see how we can clarify things but meawhile:

In your case, you are looking for a template like
`{{eventNow.custom.<name-of-your-field-in-lowercase>}}`

the easiest way to inspect the object so you can build your template, is to [poll ontime](https://docs.getontime.no/api/protocols/http/#get-ontime-runtime-state), the response is the exact object you need.
Mind you that some elements like `eventNow` are only available if something is loaded

So in short:
- load an event that contains the property you want to use
- in the browser do localhost:4001/api/poll 
- inspect the object and build your template by following the properties (note that in the case of custom fields, the key is all lower casing)

Hope this helps, feel free to continue the conversation or reach out to mail@getontime.no

**Reply by DaRealFrost** [reply]
Now that you explained it, I understand. It seems to be working, I must have just missed them examples. Thank you very much.
https://frost-dev-acc.discowd.com/chrome_HbRfDi46Dk.png

**Reply by cpvalente** [reply]
No worries, if you have any suggestions for how the documentation could have helped you or should otherwise be improved do let me know
